src/components/fiter_box/constants.js

```diff
@@ -0,0 +1,11 @@
+import { t } from '../../translations';
+
+export const access_types = [[1, 'PUBLIC', t.public], [2, 'PRIVATE', t.private]];
+export const plug_types = [
+  [1, 'Type2Mennekes', 'Type 2 Mennekes'],
+  [2, 'Type 3A', 'Type 3A'],
+  [3, 'CHAdeMO', 'CHAdeMO'],
+  [4, 'CCS', 'Type 1 CCS'],
+  [5, 'Schuko', 'Schuko'],
+  [6, 'Type2 - 230Vac', 'Type2 - 230Vac']
+];
```

Comment:
What happens here, if the API gives a new type, that is not within this list? Is there a fallback for new types?

Reply:
@Piiit  How can I get them from the new API?

Reply:
@fedriz You could just take all station-metadata outlets and then within your application go through that list of json objects and filter out all plug types.

```
ninja/api/v2/flat/EChargingPlug?limit=-1&offset=0&select=smetadata.outlets&where=sactive.eq.true&shownull=false&distinct=true
```

At the moment it is not possible to select only the certain attributes within an object of all list elements, like it would be possible with `xpath` or similar libraries. That needs to be done in your business logic.

As future API questions may arise, here some hints:
- Query the whole tree first, just checking for your needed stationtype `EChargingStation` or `EChargingPlug`, use only active stations `where=sactive.eq.true`
- then have a look at the content and think of what you need, select that with `select=....`
- This API is different from a standard Restful API, hence you need to think more like a SQL statement about it

Cheers,
Peter

Reply:
Ok, thank you, I'll try this approach

Reply:
@Piiit many results in `/flat/EChargingStation,EChargingPlug?limit=-1&offset=0&select=smetadata&where=sactive.eq.true&shownull=false&distinct=true` don't have any `outlet`, is it normal? For the time being I will use only the ones with the property

Reply:
@fedriz Yes that is clear, because only plugs have outlets. The hierarchy is as follows: 
```
station
   `--- plug
           `---- outlet
```

Reply:
I've realized now that using EChargingStation,EChargingPlug the results are not nested by hierarchy